v2/src/actors/cal_get_promise.js

```diff
@@ -1,5 +1,3 @@
-var XMLHttpRequest = new XMLHttpsRequest();
```

Comment:
This line was causing two problems:

1. This line was reassigning `XMLHttpRequest` to be an instance of the non-exist class `XMLHttpsRequest`. (It's `XMLHttpRequest`, see the other comment below.)
2. Trying to call `new` on an instance instead of a class will not work.